load/run_load.go

```diff
@@ -326,7 +327,30 @@ func (load *AppLoad) runRequest(tasks *[]Task, client *http.Client) {
 			// For each goroutine we want to reuse a buffer for performance reasons.
 			bodyBuffer := make([]byte, 50000)
 			load.HandlerParams.sendTraffic.Add(1)
+
+			// TODO: rolling read request data
```

Comment:
Why TODO? Isn't this what you're adding here?

Reply:
sorry, just forget to remove this comment